tests/epyccel/test_arrays_multiple_assignments.py

```diff
@@ -301,6 +302,22 @@ def f(b1 : bool, b2 : bool):
     assert f(True,False) == f2(True,False)
     assert f(False,True) == f2(False,True)
 
+def test_conda_flag_disable(language):
+    def one():
+        return True;
+    with pytest.warns(None) as record1:
+        f1 = epyccel(one, language='c', is_conda_warnings_disabled = True)
+    assert len(record1) == 0 # Equals 0 on every platform
+
+def test_conda_flag_verbose(language):
+    def one():
+        return True;
+    with pytest.warns(None) as record1:
+        epyccel(one, language='c', is_conda_warnings_detailed = True)
+    if len(record1)>0:
+        warn_message = record1[1].message
```

Comment:
yes
but record is an array of warnings catched by pytest.
1. is the see doc.
2. conda ignored path
3. are the paths.
in my env there are 4 warning raised.

Reply:
Normally i should put the index 2, cause there where the path of conda ignored are listed, cause record is an array so i need to compare the warning where the paths are listed
```python
    def one():
        return True;
    with pytest.warns(None) as record1:
        epyccel(one, language='c', is_conda_warnings_detailed = True)
    if len(record1)>0:
        warn_message = record1[len(record1)-1].message
        assert str(warn_message) in os.environ['PATH']
```

Reply:
Ah! Sorry I know why Windows doesn't like it. It's because the warning that it is generating is a different one about compilation. You will probably have to deactivate this one for windows too